dev/reference/sql/statements/alter-database.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I think we also need to add this file to `v3.0`?

Reply:
Added, thanks!